felix/bpf/bpf.go

```diff
@@ -387,6 +387,69 @@ type mapEntry struct {
 	Err   string   `json:"error"`
 }
 
+func (me *mapEntry) UnmarshalJSON(data []byte) error {
+	type entry struct {
+		Key   []string    `json:"key"`
+		Value interface{} `json:"value"`
+		Err   string      `json:"error"`
```

Comment:
I don't see how Err can get set, looks like value would be an object with an error inside it?

Parsing mixed type json is a pain but I might do:
```
var untyped struct {
  Key []string
  Value any
}

err := json.Unmarshal(data, &untyped)
if err != nil {
  ...
}

switch typed := untyped.Value.(type) {
  case map[string]any:
    // An error?
    err = fmt.Errorf("value error: %v", typed["error"])
  case []any:
    // A value
  ...
}
```

Reply:
```
  case map[string]any:
    // An error?
    err = fmt.Errorf("value error: %v", typed["error"])
```
I think we should not return an error when we see an error on an entry.

> I don't see how Err can get set

The output might have changed and is not longer settable. Apparently an error is delivered in a map[string]string as "error" now